app/controllers/users_controller.rb

```diff
@@ -66,4 +67,18 @@ def statistics
               owner(current_user.id).
               latest
   end
+
+  private
+
+  def user_slide_with_paginate(user_id)
```

Comment:
__[rubocop]__
* Style/IndentationWidth: Use 2 (not 0) spaces for rails indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)